scipy-2019/videos/challenges-in-detecting-physiological-changes-using-wearable-sensor-data-scipy-2019-e-preble.json

```diff
@@ -10,10 +10,18 @@
       "url": "https://www.scipy2019.scipy.org/confschedule"
     }
   ],
-  "speakers": [],
+  "speakers": [
+    "Maria Davila-Hernandez",
+    "Kristin Gilchrist",
+    "Meghan Hegarty-Craver",
+    "Michele Hernandez",
+    "Gregory Lewis",
+    "Edward Preble",
```

Comment:
Seems that Edward Preble is the speaker and the rest are collaborators not seen in the video.

Reply:
Ok, now its only Edward Preble. Thanks for noticing that.